main.md

```diff
@@ -579,30 +582,33 @@ This document introduces the following client registration metadata
 DPoP when requesting tokens from the authorization server.
 
 `dpop_bound_access_tokens`
-:   Boolean value specifying whether the client always uses DPoP for token requests.  If omitted, the default value is `false`.
+:   Boolean value specifying whether the client is required to issue DPoP token requests.  If omitted, the default value is `false`.
```

Comment:
"Always uses" is clearer than the proposed replacement text.  Please retain wording along those lines.